styleguide/_themes/derek/scss/_main.scss

```diff
@@ -5,7 +5,7 @@
 @import 'globalElements/mainnav';
 @import 'globalElements/rug';
 @import 'globalElements/footer';
-@import 'globalElements/basement';
+@import '../../../derek/global-components/basement/basement';
```

Comment:
If you mean, doing something like this
```
@import '#{$baseUrl}/component';
```
SASS won't let you use a variable inside an import